docs/tutorials/t2hahn_characterization.ipynb

```diff
@@ -0,0 +1,507 @@
+      "text/plain": [
+       "<Figure size 576x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "exp_with_p0 = T2Hahn(qubit, delays)\n",
+    "exp_with_p0.set_analysis_options(p0={\"amp\": 0.5, \"tau\": 20 * conversion_factor, \"base\": 0.5})\n",
+    "expdata_with_p0 = exp_with_p0.run(backend=backend, shots=2000)\n",
+    "expdata_with_p0.block_for_results()\n",
+    "\n",
+    "# Display fit figure\n",
+    "display(expdata_with_p0.figure(0))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "DbAnalysisResultV1\n",
+      "- name: @Parameters_T2HahnAnalysis\n",
+      "- value: [4.78978431e-01 5.02409209e-01 2.01192655e-05] ± [5.09032092e-03 3.07792331e-03 5.78387141e-07]\n",
+      "- χ²: 0.5509343846546172\n",
+      "- quality: good\n",
+      "- extra: <4 items>\n",
+      "- device_components: ['Q0']\n",
+      "- verified: False\n",
+      "DbAnalysisResultV1\n",
+      "- name: T2\n",
+      "- value: 2.011926549231594e-05 ± 5.783871411742618e-07 s\n",
+      "- χ²: 0.5509343846546172\n",
+      "- quality: good\n",
+      "- device_components: ['Q0']\n",
+      "- verified: False\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Print results\n",
+    "for result in expdata_with_p0.analysis_results():\n",
+    "    print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Number of echoes\n",
+    "The user can provide the number of echoes that the circuit will perform. This will determine the amount of delay and echo gates. As the number of echoes increase, the total time of the circuit will grow. The echoes decrease the effects of $T_{1}$ noise and frequency inaccuracy estimation. Due to that, Hahn Echo experiment let us estimate $T_{2}$ better.\n",
```

Comment:
```suggestion
    "The user can provide the number of echoes that the circuit will perform. This will determine the amount of delay and echo gates. As the number of echoes increases, the total time of the circuit will grow. The echoes decrease the effects of $T_{1}$ noise and frequency inaccuracy estimation. Due to that, the Hahn Echo experiment improves our estimate for $T_{2}$.\n",
```

Reply:
Changed as suggested.